lab_1_keywords_tfidf/main.py

```diff
@@ -33,7 +80,13 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    not_stop_words = []
+    if correct_list(tokens, str, "no") is True and correct_list(stop_words, str, "yes") is True:
```

Comment:
again - reverse the condition and return None as soon as posible

Reply:
and take a look at https://stackoverflow.com/questions/39983695/what-is-truthy-and-falsy-how-is-it-different-from-true-and-false
It helps remove `is True` from conditions